Bluetooth: Mesh: Fix dim to dark behavior (#11903)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`bluetooth_mesh_light_lc` sample:

  * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.

>   * Fixed an issue where the sample could return an invalid Light Lightness Status message if the transition time was evaluated to zero.

Reviewer: Looks good, just consider moving this bullet point before the one that starts with Removed. The order of changes is:
- Added
- Updated
- Fixed
- Removed

  * Updated:

    * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option, that is controlling timeout for storing of model states, is replaced by the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option.

>     * The Light Lightness Actual and Generic Power Level states of the :ref:`bt_mesh_lightness_srv_readme` and :ref:`bt_mesh_plvl_srv_readme` models can not dim to off as a result of binding with Generic Level state when receiving Generic Delta Set and Generic Move Set messages.

Reviewer: ```suggestion
    * The Light Lightness Actual and Generic Power Level states of the :ref:`bt_mesh_lightness_srv_readme` and :ref:`bt_mesh_plvl_srv_readme` models cannot dim to off.
      This is due to binding with Generic Level state when receiving Generic Delta Set and Generic Move Set messages.
```
